cw-relayer/relayer/relayer.go

```diff
@@ -180,7 +183,8 @@ func generateContractRelayMsg(forceRelay bool, requestID uint64, resolveTime int
 		RequestID:   requestID,
 	}
 
-	factor := types.NewDec(10).Power(18)
+	// base (USD) representation in contract
+	factor := types.NewDec(10).Power(9)
```

Comment:
Let's turn this into a variable at the top of this file and make it more clear.

```suggestion
	// factor is used to convert prices into the correct denom for contracts
	factor := types.NewDec(10).Power(9)
```